cinema/models.py

```diff
@@ -35,12 +38,20 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_path(movie: "Movie", filename: str) -> pathlib.Path:
+    filename = (
+        f"{slugify(movie.title)}-{uuid.uuid4()}{pathlib.Path(filename).suffix}"
+    )
+    return pathlib.Path("upload/movies/") / pathlib.Path(filename)
```

Comment:
The `movie_image_path` function is correctly defined to generate a unique file path for movie images using the movie title and a UUID. Ensure that the `slugify` function is imported and used correctly to handle special characters in movie titles.